torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -1470,35 +1612,38 @@ void CudaFuseGraph(std::shared_ptr<Graph>& graph) {
   // dependency and add inputs to conditional constant generated by
   // aten::profile_ivalue
   traverseProfileIValues(graph->block(), ExtractProfileIValue);
-  GRAPH_DUMP("insert conditional constant from profile_ivalue: ", graph);
+  GRAPH_DEBUG("insert conditional constant from profile_ivalue: ", *graph);
 
   // TODO: we need to properly restore shape information after fusion.
   // shamelessly use tool from NNC.
   RemoveProfileNodesAndSpecializeTypes(graph);
-  GRAPH_DUMP("After Profiling Nodes Removed: ", graph);
+  GRAPH_DEBUG("After Profiling Nodes Removed: ", *graph);
 
   markMissingType(graph->block());
-  GRAPH_DUMP("After mark missing type: ", graph);
+  GRAPH_DEBUG("After mark missing type: ", *graph);
 
   // TODO: separate passes into different file;
   // TODO: restore decomposition after fusion, in case we are decomposing
   //       operation that can't be fused;
   decomposeLinearOps(graph->block());
-  GRAPH_DUMP("decompose operations by nvfuser: ", graph);
+  GRAPH_DEBUG("decompose operations by nvfuser: ", *graph);
 
   CudaGraphFuser(graph->block(), graph).run();
-  GRAPH_DUMP("After Fusion: ", graph);
+  GRAPH_DEBUG("After Fusion: ", *graph);
 
   // guard input types as well as conditional constants from
   // aten::profile_ivalue
   guardFusionGroups(graph->block());
-  GRAPH_DUMP("After Guard Fusion: ", graph);
+  GRAPH_DEBUG("After Guard Fusion: ", *graph);
+
+  alterBatchNormImpls(graph->block());
```

Comment:
these are just necessary graph mutation to make PyTorch BN graph to be compatible with nvfuser.